### PR TITLE
[INLONG-8300][Manager] Support previewing data of Pulsar

### LINKED ISSUE #8300: [Feature][Manager] Support previewing data of Pulsar
### Description

Supports data preview function for pulsar.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8300 

### Motivation

Supports data preview function for pulsar.

### Modifications

Supports data preview function for pulsar.

### CODE REVIEW

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/consume/DisplayMessage.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+/**
+ * Inlong display message info
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("Inlong display message info")
+public class DisplayMessage {
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/consume/DisplayMessage.java]: done.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java [hunk 2]
```diff
@@ -73,4 +74,8 @@ Call<Response<InlongStreamInfo>> getStream(@Query("groupId") String groupId,
 
     @POST("stream/parseFields")
     Call<Response<List<StreamField>>> parseFields(@Body ParseFieldRequest parseFieldRequest);
+
+    @GET("stream/queryMessage")
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/service/InlongStreamApi.java]: done.

File: inlong-common/src/main/java/org/apache/inlong/common/enums/MessageWrapType.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+ */
+
+package org.apache.inlong.common.enums;
+
+/**
+ * Enum of inlong component type.
+ */
+public enum MessageWrapType {
+
+    NONE(0, "NONE"),
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-common/src/main/java/org/apache/inlong/common/enums/MessageWrapType.java:25]: done.

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java [hunk 4]
```diff
@@ -183,4 +183,10 @@ public class InlongConstants {
      */
     public static final String BATCH_PARSING_FILED_JSON_COMMENT_PROP = "desc";
 
+    private static final int COMPRESS_TYPE_NONE = 0;
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/consts/InlongConstants.java]: done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java [hunk 5]
```diff
@@ -71,4 +76,10 @@ default void createQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo st
     default void deleteQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo streamInfo, String operator) {
     }
 
+    default List<DisplayMessage> queryLastestMessage(InlongGroupInfo groupInfo, InlongStreamInfo streamInfo,
+            Integer messageNumber)
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java]: done.

**Review by vernedeng — APPROVED** [review]

File: inlong-common/src/main/java/org/apache/inlong/common/enums/MessageWrapType.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    PB(1, "PB"),
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-common/src/main/java/org/apache/inlong/common/enums/MessageWrapType.java]: Have been changed to use `DataProxyMsgEncType.MSG_ENCODE_TYPE_PB`.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java [hunk 7]
```diff
@@ -61,6 +64,8 @@ public class PulsarResourceOperator implements QueueResourceOperator {
      */
     public static final String PULSAR_SUBSCRIPTION = "%s_%s_%s_consumer_group";
 
+    public static final String PULSAR_SUBSCRIPTION_REAL_TIME_REVIEW = "%s_%s_consumer_group_real_time_review";
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java:67]: ->
```
PULSAR_SUBSCRIPTION_REALTIME_REVIEW = "%s_%s_consumer_group_realtime_review"
```

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java [hunk 8]
```diff
@@ -71,4 +76,10 @@ default void createQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo st
     default void deleteQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo streamInfo, String operator) {
     }
 
+    default List<BriefMQMessage> queryLastestMessage(InlongGroupInfo groupInfo, InlongStreamInfo streamInfo,
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java:79]: More javadoc, please.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java [hunk 9]
```diff
@@ -71,4 +76,10 @@ default void createQueueForStream(InlongGroupInfo groupInfo, InlongStreamInfo st
...
+            Integer messageCount)
+            throws PulsarClientException {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/QueueResourceOperator.java:81]: One line is enough.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java (hunk 7 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java]: Done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+import static org.apache.inlong.sort.formats.inlongmsg.InLongMsgUtils.INLONGMSG_ATTR_ENTRY_DELIMITER;
+import static org.apache.inlong.sort.formats.inlongmsg.InLongMsgUtils.INLONGMSG_ATTR_KV_DELIMITER;
+import static org.apache.inlong.sort.formats.inlongmsg.InLongMsgUtils.INLONGMSG_ATTR_TIME_DT;
+import static org.apache.inlong.sort.formats.inlongmsg.InLongMsgUtils.INLONGMSG_ATTR_TIME_T;
+
+@Slf4j
+@Service
+public class InlongMsgDeserializeOperator implements DeserializeOperator {
+
+    public static final String NODE_IP = "NodeIP";
```
[the remark this answers is not in the document]
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/InlongMsgDeserializeOperator.java]: Done.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/DeserializeOperator.java [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    // keys in attributes
+    String INLONGMSG_ATTR_TIME_T = "t";
+    String INLONGMSG_ATTR_TIME_DT = "dt";
+
+    /**
+     * Determines whether the current instance matches the specified type.
+     */
+    boolean accept(DataProxyMsgEncType type);
+
+    default List<BriefMQMessage> decodeMsg(InlongStreamInfo streamInfo,
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/message/DeserializeOperator.java:47]: javadoc here~

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java [hunk 12]
```diff
@@ -299,4 +304,32 @@ private void deletePulsarTopic(InlongPulsarInfo pulsarInfo, PulsarClusterInfo pu
         }
     }
 
+    public List<BriefMQMessage> queryLastestMessage(InlongGroupInfo groupInfo,
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarResourceOperator.java:307]: javadoc here~

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java [hunk 13]
```diff
@@ -374,4 +385,40 @@ private boolean subscriptionExists(PulsarAdmin pulsarAdmin, String topic, String
         return false;
     }
 
+    /**
+     * Query topic message for the given pulsar cluster.
+     */
+    public List<BriefMQMessage> queryLastMessage(PulsarAdmin pulsarAdmin, String topicFullName, String subName,
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/queue/pulsar/PulsarOperator.java:391]: Should be `Lastest`?

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**